app/view/show.py

```diff
@@ -196,8 +202,8 @@ def get_group_data(avg):
         'versions': versions,
         'issues': issues,
         'issue_types': issue_types,
-        'advisories': published_advisories,
-        'advisory_pending': advisory_pending
+        'advisories': advisories,
+        'advisories_pending': advisory_pending
```

Comment:
would it make sense to also replace the variable `advisory_pending` to `advisories_pending` for consistency